packages/components/src/internal/components/entities/DataTypeSelector.tsx

```diff
@@ -194,73 +335,19 @@ export const DataTypeSelector: FC<DataTypeSelectorProps> = memo(props => {
-                        return (
-                            <li key={entityId} className="folder-faceted-data-type">
-                                <div className="form-check">
-                                    <input
-                                        className="form-check-input filter-faceted__checkbox"
-                                        type="checkbox"
-                                        name={'field-value-' + index}
-                                        onChange={event => onChange(entityId, false, event.target.checked)}
-                                        checked={uncheckedEntities?.indexOf(entityId) < 0}
-                                        disabled={disabled}
-                                    />
-                                    <div
-                                        className="margin-left-more folder-datatype-faceted__value"
-                                        onClick={() => onChange(entityId, true)}
-                                    >
-                                        {type.labelColor && (
-                                            <ColorIcon
-                                                cls="label_color color-icon__circle-small"
-                                                value={type.labelColor}
-                                            />
-                                        )}
-                                        {type.label}
-                                    </div>
-                                </div>
-                                {!!type.sublabel && <div className="help-block margin-left-more">{type.sublabel}</div>}
-                                {showUncheckedWarning && _getUncheckedEntityWarning(entityId)}
-                            </li>
-                        );
-                    })}
-                </ul>
-            );
-        },
-        [uncheckedEntities, disabled, showUncheckedWarning, _getUncheckedEntityWarning, onChange]
-    );
-
-    // FIXME: this should be a component, not a callback
-    const getEntitiesList = useCallback((): React.ReactNode => {
-        if (!columns || columns === 1) {
-            return <div className="col-xs-12">{getEntitiesSubList(dataTypes)}</div>;
-        }
-
-        const lists = [];
-        const colWidth = 12 / columns;
-        const subSize = Math.ceil(dataTypes.length / columns);
-        for (let i = 0; i < columns; i++) {
-            const maxInd = Math.min(dataTypes.length, (i + 1) * subSize);
-            const subList = dataTypes.slice(i * subSize, maxInd);
-            lists.push(
-                <div className={`col-xs-12 col-md-${colWidth}`} key={i}>
-                    {getEntitiesSubList(subList)}
-                </div>
-            );
-        }
-        return <>{lists}</>;
-    }, [dataTypes, columns, getEntitiesSubList]);
+    const generateInactiveSectionHeader = useMemo(() => {
```

Comment:
```suggestion
    const inactiveSectionHeader = useMemo(() => {
```
Use of `generate` seems odd to me when this is really most likely to be static.